libre-2/css/editor-blocks.css

```diff
@@ -1,535 +0,0 @@
-
-/*--------------------------------------------------------------
-1.0 General Typography
---------------------------------------------------------------*/
-
-.edit-post-visual-editor .editor-block-list__block,
-.editor-default-block-appender textarea.editor-default-block-appender__content {
-	font-family: "Libre Baskerville", Libre, Georgia, Times, serif;
-}
-
-.edit-post-visual-editor .editor-block-list__block,
-.edit-post-visual-editor .editor-block-list__block p,
-.editor-default-block-appender textarea.editor-default-block-appender__content {
-	font-size: 16px;
-}
-
-.edit-post-visual-editor .editor-block-list__block {
-	color: #404040;
-}
-
-/* Post title */
-
-.editor-post-title__block .editor-post-title__input {
-	color: #404040;
-	font-family: "Libre Baskerville", Libre, Georgia, Times, serif;
-	font-size: 29px;
-}
-
-@media screen and ( min-width: 720px ) {
-	.editor-post-title__block .editor-post-title__input {
-		font-size: 47px;
-	}
-}
-
-/* Headings */
-
-.edit-post-visual-editor h1,
-.wp-block-freeform.block-library-rich-text__tinymce h1 {
-	font-size: 29px;
-}
-
-.edit-post-visual-editor h2,
-.wp-block-freeform.block-library-rich-text__tinymce h2 {
-	font-size: 23px;
-}
-
-.edit-post-visual-editor h3,
-.wp-block-freeform.block-library-rich-text__tinymce h3 {
-	font-size: 18px;
-}
-
-.edit-post-visual-editor h4,
-.wp-block-freeform.block-library-rich-text__tinymce h4 {
-	font-size: 14px;
-}
-
-.edit-post-visual-editor h5,
-.wp-block-freeform.block-library-rich-text__tinymce h5 {
-	font-size: 13px;
-}
-
-.edit-post-visual-editor h6,
-.wp-block-freeform.block-library-rich-text__tinymce h6 {
-	font-size: 12px;
-}
-
-@media (min-width: 720px) {
-	.editor-styles-wrapper.edit-post-visual-editor h1,
-	.wp-block-freeform.block-library-rich-text__tinymce h1 {
-		font-size: 47px;
-	}
-
-	.editor-styles-wrapper.edit-post-visual-editor h2,
-	.wp-block-freeform.block-library-rich-text__tinymce h2 {
-		font-size: 29px;
-	}
-
-	.editor-styles-wrapper.edit-post-visual-editor h3,
-	.wp-block-freeform.block-library-rich-text__tinymce h3 {
-		font-size: 23px;
-	}
-
-	.editor-styles-wrapper.edit-post-visual-editor h4,
-	.wp-block-freeform.block-library-rich-text__tinymce h4 {
-		font-size: 18px;
-	}
-
-	.editor-styles-wrapper.edit-post-visual-editor h5,
-	.wp-block-freeform.block-library-rich-text__tinymce h5 {
-		font-size: 14px;
-	}
-
-	.editor-styles-wrapper.edit-post-visual-editor h6,
-	.wp-block-freeform.block-library-rich-text__tinymce h6 {
-		font-size: 13px;
-	}
-}
-
-/*--------------------------------------------------------------
-2.0 General Block Settings
---------------------------------------------------------------*/
-
-/* Main content width */
-
-.wp-block {
-	max-width: 705px; /* 675px + 30px to account for padding */
-}
-
-/* Link styles */
-
-.edit-post-visual-editor a,
-.editor-block-list__block a,
-.wp-block-freeform.block-library-rich-text__tinymce a {
-	box-shadow: 0 1px 0 0 currentColor;
```

Comment:
This has been lost from the editor, but before your change, it wasn't exactly like the frontend either... It's just that the frontend targets `.entry-content a` so that will never work on the editor